### PR TITLE
Add Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by ahdg6**
Add Simplified Chinese Translation

### CODE REVIEW

**Comment by CDFN** [comment]
In my opinion issue templates & badge should go to other pull requests so we can discuss them there.

**Comment by ahdg6** [comment]
> 
> 
> In my opinion issue templates & badge should go to other pull requests so we can discuss them there.

Thank you for your sugeestion. now separated.

File: README.md [hunk 1]
```diff
@@ -1,30 +1,5 @@
...
-</div>
-
-## Table of Contents
-* [Introduction](#introduction)
-  * [Design](#design)
-  * [Features](#features)
-* [Basic Info](#basic-info)
-* [Developer API](#developer-api)
+# LibertyBans
+The be-all, end-all of discipline.
```
Comment (CDFN) [inline README.md]: I think that change should be separated aswell?

Comment (CDFN) [reply README.md]: Nevermind, looked at wrong commit :+1:

File: bans-core/src/main/resources/lang/messages_zh.yml [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+    fragments:
+      MONTHS: '%VALUE% 月'
+      HOURS: '%VALUE% 时'
+      YEARS: '%VALUE% 年'
+      MINUTES: '%VALUE% 分'
+      WEEKS: '%VALUE% 周'
+      DAYS: '%VALUE% 天'
+
+
+
```
Comment (CDFN) [inline bans-core/src/main/resources/lang/messages_zh.yml:379]: Leave only one newline at the end of file

**Pull request merged**